src/datasources/track-api.js

```diff
@@ -1,33 +1,33 @@
-const { RESTDataSource } = require('apollo-datasource-rest');
+const { RESTDataSource } = require("apollo-datasource-rest");
```

Comment:
We should really pick a convention lol. It looks like Odyssey code snippets force double quotes so maybe let's start to go towards that style.